### PR TITLE
add a release workflow to publish to galaxy

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v*
```
Comment (evgeni) [inline .github/workflows/release.yml]: can we check if this is a number?

**Comment by evgeni** [comment]
@ehelms want to have the honor here? :)

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
```
Comment (mdellweg) [inline .github/workflows/release.yml]: :+1:

Comment (ehelms) [reply .github/workflows/release.yml]: I never have liked to the little `v` in the tags but looking at releases that ship has sailed.

**Comment by ehelms** [comment]
I don't suppose we could push a version bump and it could do the tag and release for us?

**Comment by evgeni** [comment]
> I don't suppose we could push a version bump and it could do the tag and release for us?

actually, we could by patch matching the galaxy.yml and then doing some magic. but I prefer to do the tags by hand (and sign them).

**Pull request merged**